lib/docker.sh

```diff
@@ -4,7 +4,7 @@
 
 # Test if docker container exist, might be running or stopped
 function container_exists() {
-    [[ "$(docker ps -a -q -f name=^/"${1}"\$)" ]]
+    [[ $(docker_is_installed) ]] && [[ "$(docker ps -a -q -f name=^/"${1}"\$)" ]]
```

Comment:
we should add this guard to all the functions in this file, that expect the docker command